configs/config.toml

```diff
@@ -1,3 +1,6 @@
+[log]
+filter = "xain_fl=debug,info"
```

Comment:
i thought that higher priority levels would automatically include lower priority levels when logging, like `error > warn > info > debug > trace` for the [level filters](https://docs.rs/tracing-subscriber/0.2.6/tracing_subscriber/filter/struct.LevelFilter.html). is that not the case?

Reply:
That is right. However, a filter allows us to define multiple logging directives. 
So `xain_fl=debug` means: turn on `debug` logging for the `xain_fl` module and 
`info` means turn on `info` logging for all modules.

Reply:
ahh true, i missed that only the first level after the filter actually applies to the filter.